services/housetables/src/main/java/com/linkedin/openhouse/housetables/api/spec/model/UserTable.java

```diff
@@ -61,6 +61,22 @@ public class UserTable {
   @JsonProperty(value = "creationTime")
   private Long creationTime;
 
+  @Schema(
+      description =
+          "Timestamp in milliseconds when the table was soft-deleted. "
+              + "If null, then the table is active",
+      example = "1751907524",
+      defaultValue = "null")
+  private Long deletedAtMs;
+
+  @Schema(
+      description =
+          "Unix timestamp in milliseconds indicating when the table will be deleted permanently by another process. "
+              + "If the current time is past the TTL, then the table could be arbitrarily deleted at any time.",
+      example = "1751907524",
+      defaultValue = "null")
+  private Long purgeAfterMs;
```

Comment:
Should this be `purgeAfterMs` or `purgeFromMs`? At API level `purgeFromMs` is used. So should be good to use one format consistently.

Reply:
Good point, I think I will keep everything to `purgeAfterMs`, it's because on the API side it seems like I want to purge all datasets before <x> time, but at an entity level the entity describes itself to purge after <x> time. I think keeping it consistent is easier to reason about though.